scripts/clone-multisig.ts

```diff
@@ -0,0 +1,44 @@
+import hre from 'hardhat';
+import { DeploymentManager } from '../plugins/deployment_manager/DeploymentManager';
+import { SafeFactory } from '@safe-global/safe-core-sdk';
+import EthersAdapter from '@safe-global/safe-ethers-lib';
+
+const SRC_NETWORK = process.env['SRC_NETWORK'] ?? 'mainnet';
+const DST_NETWORK = process.env['DST_NETWORK'] ?? 'hardhat';
+
+async function main() {
+  await hre.changeNetwork(SRC_NETWORK);
+
+  const dm = new DeploymentManager(SRC_NETWORK, 'usdc', hre);
+
+  const signer_ = await dm.getSigner();
+  const comet = await dm.contract('comet');
+  const guardian = await comet.pauseGuardian();
+
+  // Get owners and threshold from existing multisig
+  const GnosisABI = [
+    {"constant":true,"inputs":[],"name":"getOwners","outputs":[{"internalType":"address[]","name":"","type":"address[]"}],"payable":false,"stateMutability":"view","type":"function"},
+    {"constant":true,"inputs":[],"name":"getThreshold","outputs":[{"internalType":"uint256","name":"","type":"uint256"}],"payable":false,"stateMutability":"view","type":"function"},
+  ];
+  const GnosisSafeContract = new hre.ethers.Contract(guardian, GnosisABI, hre.ethers.provider);
+  const owners = await GnosisSafeContract.getOwners();
+  const threshold = await GnosisSafeContract.getThreshold();
```

Comment:
Should we check that these are non-zero values, just to check that the `pauseGuardian` is indeed a gnosis multi-sig?

Reply:
These calls will revert if its not a multisig. It could be empty but it also prints out what config so you can see 🤷